dist-git.spec

```diff
@@ -114,6 +115,12 @@ cp -a configs/httpd/dist-git.conf %{buildroot}%{_sysconfdir}/httpd/conf.d/
 cp -a configs/httpd/dist-git/* %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git/
 cp -a configs/systemd/*        %{buildroot}%{_unitdir}/
 
+# ------------------------------------------------------------------------------
+# python package
+# ------------------------------------------------------------------------------
+mkdir -p %{buildroot}%{python3_sitelib}
```

Comment:
I think we would like to support both python3 and python2. On CentOS7 (and lower) python2 is used (shebang in upload.cgi is substituted for python2).

Reply:
True, I will update it